src/portal/src/app/base/project/repository/artifact/artifact-list-page/artifact-list/artifact-list-tab/artifact-list-tab.component.ts

```diff
@@ -1108,7 +1144,7 @@ export class ArtifactListTabComponent implements OnInit, OnDestroy {
               if (res.headers) {
                 let xHeader: string = res.headers.get("x-total-count");
                 if (xHeader) {
-                  item.tagNumber = Number.parseInt(xHeader);
+                  item.tagNumber = Number.parseInt(xHeader, 10);
```

Comment:
If the `xHeader` is not a number?

Reply:
I think the back-end API  should handle this.  if x-total-count is set, it should be a number bigger than zero